scripts/create_test

```diff
@@ -633,6 +633,7 @@ def _main_func(description):
 
     success = False
     run_count = 0
+    os.environ["FROM_CREATE_TEST"] = "True"
```

Comment:
you can remove this

Reply:
Good catch-- this is not needed because the test_scheduler.py change will catch the error.

Reply:
I removed the extra line in create_test.